Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButtonHoloLens2.cs

```diff
@@ -15,6 +15,30 @@ public class PressableButtonHoloLens2 : PressableButton
         [Tooltip("The icon and text content moving inside the button.")]
         private GameObject movingButtonIconText = null;
 
+        [SerializeField]
+        [Tooltip("The visuals which become compressed (scaled) along the z-axis when pressed.")]
+        private GameObject compressableButtonVisuals = null;
+        public GameObject CompressableButtonVisuals
+        {
+            get => compressableButtonVisuals;
+            set
+            {
+                compressableButtonVisuals = value;
+
+                if (compressableButtonVisuals != null)
+                {
+                    initialCompressableButtonVisualsLocalScale = compressableButtonVisuals.transform.localScale;
+                }
+
+            }
+        }
+
+        [SerializeField]
+        [Range(0.0f, 1.0f)]
+        [Tooltip("The minimum percentage of the original scale the compressableButtonVisuals can be compressed to.")]
```

Comment:
MinCompressPercentage  is a public field: please add this tooltip as a /// comment as well so it shows up in the docs

Reply:
Good catch, I'll fix that!